internal/cluster/distribution/eks/eksworkflow/workflow_update_node_pool.go

```diff
@@ -98,16 +98,19 @@ func (w UpdateNodePoolWorkflow) Execute(ctx workflow.Context, input UpdateNodePo
 		activityOptions.RetryPolicy = &cadence.RetryPolicy{
 			InitialInterval:    10 * time.Second,
 			BackoffCoefficient: 1.01,
+			MaximumAttempts:    10,
 			MaximumInterval:    10 * time.Minute,
 		}
 
 		var output CalculateNodePoolVersionActivityOutput
 
+		processActivity := process.StartActivity(ctx, CalculateNodePoolVersionActivityName)
```

Comment:
I'm not sure this is necessary. I actually plan to move version calculation out of the workflow.